requirements/base.in

```diff
@@ -1,5 +1,6 @@
+-e git+https://github.com/gridsingularity/d3a-interface@update_dependencies#egg=d3a_interface
```

Comment:
we should Pin the master branch

Reply:
Yes, would pin the master branch as soon the corresponding d3a-interface PR is merged